### PR TITLE
Add LogCommand function to log used commands to a specific channel

### PR DESCRIPTION
**Pull request by OTCompa**
Dependent on #5

### CODE REVIEW

**Comment by OTCompa** [comment]
From the issue in #2:
> let's double check that it does not send ephemeral messages
The `LogCommand` function shouldn't send any ephemeral messages at all, it will only send an embed to the specified log channel. Ephemeral responses are sent by the slash command function. (Correct me if I'm misunderstanding something)

**Comment by brtran4** [comment]
> From the issue in #2:
> 
> > let's double check that it does not send ephemeral messages
> 
> The `LogCommand` function shouldn't send any ephemeral messages at all, it will only send an embed to the specified log channel. Ephemeral responses are sent by the slash command function. (Correct me if I'm misunderstanding something)

as long as you've tested and the embed isn't ephemeral then that looks good to me :)

File: main.go [hunk 1]
```diff
@@ -18,7 +18,8 @@ func main() {
 	discordToken = strings.TrimSpace(discordToken)
 
 	d := &discord.Discord{
-		Token: discordToken,
+		Token:           discordToken,
+		ModLogChannelID: "",
```
Comment (brtran4) [inline main.go:22]: let's also move this to https://github.com/naurffxiv/moddingway/pull/5

**Review by brtran4 — CHANGES_REQUESTED** [review]

File: internal/discord/command-handlers.go [hunk 2]
```diff
@@ -46,6 +51,68 @@ func (d *Discord) AddCommands(s *discordgo.Session, event *discordgo.Ready) {
...
+		d.ModLogChannelID,
+		&discordgo.MessageEmbed{
+			Author: &discordgo.MessageEmbedAuthor{
+				Name:    i.Member.User.Username,
+				IconURL: i.Member.AvatarURL(""),
+			},
+			Description: actionDescription,
+			Fields:      embedFields,
+			Timestamp:   time.Now().Format(time.RFC3339),
+		},
```
[the remark this answers is not in the document]
Comment (OTCompa) [reply internal/discord/command-handlers.go:112]: image

**Review by brtran4 — APPROVED** [review]

**Pull request merged**